packages/neuron-ui/.eslintrc.js

```diff
@@ -67,6 +67,8 @@ module.exports = {
     "react/jsx-filename-extension": [1, {
       "extensions": [".ts", ".tsx"]
     }],
+    "jsx-a11y/click-events-have-key-events": [0],
```

Comment:
Why disable these two rules

Reply:
This dialog should be interactive and can be dismissed by clicking modal
https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/SignAndVerify/index.tsx#L136-L152